zazu/plugins/eslint_styler.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""ESLint plugin for zazu."""
+import zazu.styler
+zazu.util.lazy_import(locals(), [
+    'subprocess',
+    'os',
+    'tempfile'
+])
+
+__author__ = "Patrick Moore"
+__copyright__ = "Copyright 2018"
+
+
+class ESLintStyler(zazu.styler.Styler):
+    """ESLint plugin for code styling."""
+
+    def style_string(self, string):
+        """Fix a string to be within style guidelines."""
+        temp = tempfile.NamedTemporaryFile(delete=False, suffix=".js")
+        temp_path = temp.name
+        args = ['eslint', '--fix'] + self.options + [temp_path]
+        temp.write(string)
+        temp.close()
+        try:
+            subprocess.check_output(args)
```

Comment:
Probably should catch OSError here to handle the case that eslint isn't installed.